alf/environments/suite_bsuite.py

```diff
@@ -1,3 +1,17 @@
+# Copyright (c) 2020 Horizon Robotics and ALF Contributors. All Rights Reserved.
```

Comment:
2020 -> 2021?

Reply:
Changed to 2021 for both suite_bsuite and suite_bsuite_test.